docs/design/add-in-icons-monoline.md

```diff
@@ -0,0 +1,202 @@
+
+## Office Monoline visual style
+
+The goal of the Monoline style to have consistent, clear, and accessible iconography to communicate action and features with simple visuals, ensure the icons are accessible to all users, and have a style that is consistent with those used elsewhere in Windows.
+
+The following guidelines are for 3rd party developers who want to create icons for features that will be consistent with the icons already present Office products.
+
+### Design principles
+
+-   Simple, clean, clear.
+-   Contain only necessary elements.
+-   Inspired by Windows icon style.
+-   Accessible to all users.
+
+#### Conveying meaning
+
+-   Use descriptive elements such as a page to represent a document or an envelope to represent mail.
+-   Use the same element to represent the same concept, i.e. mail is always represented by an envelope, not a stamp.
+-   Use a core metaphor during concept development.
+
+#### Reduction of Elements
+
+-   Reduce the icon to its core meaning, using only elements that are essential to the metaphor.
+-   Limit the number of elements in an icon to two, regardless of icon size.
+
+#### Consistency
+
+Sizes, arrangement, and color of icons should be consistent.
+
+#### Styling
+
+##### Perspective
+
+Monoline icons are forward-facing by default. Certain elements that require perspective and/or rotation, such as a cube are allowed, but exceptions should be kept to a minimum.
+
+##### Embellishment
+
+Monoline is a clean minimal style. Everything uses flat color, which means there are no gradients, textures, or light sources.
+
+## Designing
+
+### Sizes
+
+We recommend that you produce each icon in all these sizes to support high DPI devices. The absolutely *required* sizes are 16x, 20x, and 32x, as those are the 100% sizes.
+
+**16x, 20x, 24x, 32x, 40x, 48x, 64x, 80x, 96x**
+
+### Layout
+
+The following is an example of icon layout with a modifier.
+
+![Example of icon with modifier](../images/monolineicon1.png)  ![The same example with a grid background callouts for the base, modifier, padding and modifier cutout](../images/monolineicon2.png)
+
+#### Elements
+
+- **Base**: The main concept that the icon represents. This is usually the only visual needed for the icon, but sometimes the main concept can be enhanced with a secondary element, a modifier.
+
+- **Modifier** Any element that overlays the base; that is, a modifier that typically represents an action or a status. It modifies the base element by acting as an addition, alteration, or a descriptor.
+
+![Grid with the base area and modifier areas on it.](../images/monolineicon3.png)
+
+### Construction
+
+#### Element placement
+
+Base elements are placed in the center of the icon with in the padding. If it cannot place perfectly centered, then the base should err to the top right. In the following example, the icon is perfectly centered:
```

Comment:
```suggestion
Base elements are placed in the center of the icon within the padding. If it can't be placed perfectly centered, then the base should err to the top right. In the following example, the icon is perfectly centered:
```